spec/integration/basic/new/rails_admin_namespaced_model_new_spec.rb

```diff
@@ -27,4 +27,11 @@
       is_expected.to have_selector("textarea#cms_basic_page_content[name='cms_basic_page[content]']")
     end
   end
+
+  describe 'GET /admin/cms_basic_page/new with parameters for pre-population' do
+    it 'populates form field when corresponding parameters are passed in' do
+      visit new_path(model_name: 'cms~basic_page', cms_basic_page: {title: 'Hello'})
```

Comment:
You are visit page twice, once in before block, second time in it block. Could you fix this?